discovery-infra/test_infra/consts/consts.py

```diff
@@ -9,6 +9,8 @@
 IMAGE_NAME = "installer-image.iso"
 STORAGE_PATH = "/var/lib/libvirt/openshift-images"
 SSH_KEY = "ssh_key/key.pub"
+MASTER_TF_CPU_MODE = "host-passthrough"
```

Comment:
`host-passthrough` string is duplicated for master and worker cpu mode.
Maybe add `HOST_PASSTHROUGH_CPU_MODE` and set it to both of them? Another option is to leave only `HOST_PASSTHROUGH_CPU_MODE` and set on `conftest`?

Reply:
done